VERSION

```diff
@@ -1 +1 @@
-2.1.40
+2.2.1
```

Comment:
For now I just bumped the minor version, but need to decide what version we should use next.